sdmetrics/reports/single_table/_properties/column_shapes.py

```diff
@@ -52,9 +56,8 @@ def _generate_details(self, real_data, synthetic_data, metadata, progress_bar=No
 
             except Exception as e:
                 column_score = np.nan
-                error_message = (
-                        f'Error: {type(e).__name__} {e}'
-                )
+                error_message = f'Error: {type(e).__name__} {e}'
+                LOGGER.debug(error_message)
```

Comment:
I don't think we need to log the error message since we store the same thing on the column. I was saying I think we can log the full stack trace for debugging purposes. Since I think we may rethink our logging, for this issue I think you can just remove it

Reply:
Alright, I see, done in [c8e6cbf](https://github.com/sdv-dev/SDMetrics/pull/366/commits/c8e6cbf3c3cbef304be08a195c47c2a15fee6cc6).